guides/03_features/04_padrino-cache.html.md

```diff
@@ -7,412 +7,350 @@ title: Padrino Cache
-Padrino.cache = Padrino::Cache::Store::Redis.new(::Redis.new(:host => '127.0.0.1', :port => 6379, :db => 0))
-Padrino.cache = Padrino::Cache::Store::Memory.new(50)
-Padrino.cache = Padrino::Cache::Store::File.new(/my/cache/path)
-```
+    Padrino.cache = Padrino::Cache.new(:LRUHash) # in-memory, the default choice
+    Padrino.cache = Padrino::Cache.new(:File, :dir => Padrino.root('tmp', app_name.to_s, 'cache')) # Keeps cached values in file
+    Padrino.cache = Padrino::Cache.new(:Memcached) # Uses default server at localhost
+    Padrino.cache = Padrino::Cache.new(:Memcached, :server => '127.0.0.1:11211', :exception_retry_limit => 1)
+    Padrino.cache = Padrino::Cache.new(:Memcached, :backend => memcached_or_dalli_instance)
+    Padrino.cache = Padrino::Cache.new(:Redis) # Uses default server at localhost
+    Padrino.cache = Padrino::Cache.new(:Redis, :host => '127.0.0.1', :port => 6379, :db => 0)
+    Padrino.cache = Padrino::Cache.new(:Redis, :backend => redis_instance)
+    Padrino.cache = Padrino::Cache.new(:Mongo) # Uses default server at localhost
+    Padrino.cache = Padrino::Cache.new(:Mongo, :backend => mongo_client_instance)
 
 You can manage your cache from anywhere in your app:
 
-```ruby
-Padrino.cache = Padrino::Cache::Store::Memcache.new(::Memcached.new('127.0.0.1:11211', :exception_retry_limit => 1))
-Padrino.cache = Padrino::Cache::Store::Memcache.new(::Dalli::Client.new('127.0.0.1:11211', :exception_retry_limit => 1))
-Padrino.cache = Padrino::Cache::Store::Redis.new(::Redis.new(:host => '127.0.0.1', :port => 6379, :db => 0))
-Padrino.cache = Padrino::Cache::Store::Memory.new(50)
-Padrino.cache = Padrino::Cache::Store::File.new(/my/cache/path)
-```
-
-### Application Caching Options
+    Padrino.cache['val'] = 'test'
+    Padrino.cache['val'] # => 'test'
+    Padrino.cache.delete('val')
+    Padrino.cache.clear
 
-```ruby
-set :cache, Padrino::Cache::Store::Memcache.new(::Memcached.new('127.0.0.1:11211', :exception_retry_limit => 1))
-set :cache, Padrino::Cache::Store::Memcache.new(::Dalli::Client.new('127.0.0.1:11211', :exception_retry_limit => 1))
-set :cache, Padrino::Cache::Store::Redis.new(::Redis.new(:host => '127.0.0.1', :port => 6379, :db => 0))
-set :cache, Padrino::Cache::Store::Memory.new(50)
-set :cache, Padrino::Cache::Store::File.new(Padrino.root('tmp', app_name, 'cache') # default choice
-```
+The Padrino cache constructor `Padrino::Cache.new` calls `Moneta.new` to
+create a cache instance. Please refer to the [Moneta
+documentation](http://rubydoc.info/gems/moneta) if you have special
+requirements, for example if you want to configure the marshalling mechanism
+or use a more exotic backend.
 
-You can manage your cache from anywhere in your app:
-
-```ruby
-MyApp.cache.set('val', 'test')
-MyApp.cache.get('val') # => 'test'
-MyApp.cache.delete('val')
-MyApp.cache.flush
-```
+#### Application Caching Options
```

Comment:
Probably an artifact from the migration but would you mind dropping this down a header  level? (`###`).